templates/base.html

```diff
@@ -565,6 +565,38 @@
           transform: rotate(360deg);
         }
       }
+<<<<<<< HEAD
```

Comment:
is this something left over from a merge conflict resolution or something? Or is the <<<<<<< HEAD part of this code?